service/controller/key/machine_deployment.go

```diff
@@ -241,6 +295,17 @@ func MachineDeploymentOnDemandPercentageAboveBaseCapacity(cr infrastructurev1alp
 	return *cr.Spec.Provider.InstanceDistribution.OnDemandPercentageAboveBaseCapacity
 }
 
+func MachineDeploymentWorkerCountRatio(workers int, ratio float32) string {
```

Comment:
Do we need unit tests for this one?

Reply:
Maybe yes, I just copy/paste the code from a different file :trollface:

Reply:
done